packer/builder/azure/driver_restapi/builder.go

```diff
@@ -266,6 +271,8 @@ func (b *Builder) Run(ui packer.Ui, hook packer.Hook, cache packer.Cache) (packe
 	state.Put(constants.DiskExists, 0)
 	state.Put(constants.VmRunning, 0)
 	state.Put(constants.ImageCreated, 0)
+	state.Put(constants.Subnet, 0)
+	state.Put(constants.VNet, 0)
```

Comment:
Not sure i basically just hacked away at it until it worked. My go is still pretty weak.